res/lang/news/news_en.txt

```diff
@@ -114,64 +114,64 @@ SCORE_POINTS = points
-SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_P1 = give away 3 valuable points %MATCHKIND% to
-SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_S2 = is defeated due to blunder of keeper %TEAM1KEEPERSHORT% %MATCHKIND% against
-SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_P2 = are defeated due to blunder of keeper %TEAM1KEEPERSHORT% %MATCHKIND% against
-SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_S3 = loses with disappointed keeper %TEAM1KEEPER% %MATCHKIND% against
-SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_P3 = lose with disappointed keeper %TEAM1KEEPER% %MATCHKIND% against
-
-SPORT_SOCCER_TEAMREPORT_MATCHKIND1 = after %PLAYTIMEMINUTES% minutes of questionable play
-SPORT_SOCCER_TEAMREPORT_MATCHKIND2 = after %PLAYTIMEMINUTES% minutes of tactical duels
-SPORT_SOCCER_TEAMREPORT_MATCHKIND3 = after exciting %PLAYTIMEMINUTES% minutes of field sport
-
-SPORT_TEAMREPORT_MATCHWIN_S1 = wins thanks to %TEAM1STAR%
-SPORT_TEAMREPORT_MATCHWIN_P1 = win thanks to %TEAM1STAR%
-SPORT_TEAMREPORT_MATCHWIN_S2 = defeats %MATCHKIND%
-SPORT_TEAMREPORT_MATCHWIN_P2 = defeat %MATCHKIND%
-SPORT_TEAMREPORT_MATCHWIN_S3 = subdues
+SPORT_SOCCER_TEAMREPORT_MATCHWIN_S1 = %MATCHKIND% win, starring attacker %TEAM1STAR%, against
+SPORT_SOCCER_TEAMREPORT_MATCHWIN_P1 = %MATCHKIND% win, starring attacker %TEAM1STAR%, against
+SPORT_SOCCER_TEAMREPORT_MATCHWIN_S2 = %MATCHKIND% win, thanks to a converted corner kick by %TEAM1STARSHORT%, against
+SPORT_SOCCER_TEAMREPORT_MATCHWIN_P2 = %MATCHKIND% win, thanks to a converted corner kick by %TEAM1STARSHORT%, against
+SPORT_SOCCER_TEAMREPORT_MATCHWIN_S3 = %MATCHKIND% win, thanks to a saved penalty by keeper %TEAM1KEEPERSHORT%, against
+SPORT_SOCCER_TEAMREPORT_MATCHWIN_P3 = %MATCHKIND% win, thanks to a saved penalty by keeper %TEAM1KEEPERSHORT%, against
+SPORT_SOCCER_TEAMREPORT_MATCHWIN_S4 = %MATCHKIND% win, thanks to some brilliant saves by goalkeeper %TEAM1KEEPERSHORT%, against
+SPORT_SOCCER_TEAMREPORT_MATCHWIN_P4 = %MATCHKIND% win, thanks to some brilliant saves by goalkeeper %TEAM1KEEPERSHORT%, against
+
+SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_S1 = %MATCHKIND% give away 3 valuable points to
+SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_P1 = %MATCHKIND% give away 3 valuable points to
+SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_S2 = %MATCHKIND% are defeated, thanks to a blunder by keeper %TEAM1KEEPERSHORT%, by
+SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_P2 = %MATCHKIND% are defeated, thanks to a blunder by keeper %TEAM1KEEPERSHORT%, by
+SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_S3 = %MATCHKIND% lose, with keeper %TEAM1KEEPER% giving a disappointing performance, against
+SPORT_SOCCER_TEAMREPORT_MATCHLOOSE_P3 = %MATCHKIND% lose, with keeper %TEAM1KEEPER% giving a disappointing performance, against
+
+SPORT_SOCCER_TEAMREPORT_MATCHKIND1 = - after %PLAYTIMEMINUTES% minutes of dodgy play -
+SPORT_SOCCER_TEAMREPORT_MATCHKIND2 = - after %PLAYTIMEMINUTES% minutes of sharp tactics -
+SPORT_SOCCER_TEAMREPORT_MATCHKIND3 = - after %PLAYTIMEMINUTES% minutes of excitement -
+
+SPORT_TEAMREPORT_MATCHWIN_S1 = win, thanks to %TEAM1STAR%, against
```

Comment:
That's what I figured. When I had an actual singular form in the S1/S2/etc. the news pieces looked wrong, as a team is being treated as singular. It may be in German but in English it's much more natural to always treat a team (of playerS) as plural.